gabbytam_solution.txt

```diff
@@ -0,0 +1,93 @@
+//Problem set: https://github.com/Gabbytam/JS-Basic-Loops
+
+
+// 1. Get Even
+
+for(i=0; i<=200; i++){
+  if(i%2===0){
+    console.log(i);
+  }
+}
+
+//2. Excited Kitten
+
+i=1;
+while(i<=10){
+  console.log('Love me, pet me! HSSSSSS!');
+  let phrases= ["...human...why you taking pictures of me?...", "...the catnip made me do it...", "meow?", "...why does the red dot always get away..."];
+  let randNum= Math.floor(Math.random()*4);
+  if(i%2===0){
+    console.log(phrases[randNum]);
+  }
+  i++;
+}
+
+//3. Thermostat 
+
+let currentTemp= Math.floor(Math.random()*100)+1;
+const desiredTemp= 68;
+
+console.log('The current temp is '+ currentTemp + 'F');
+while(currentTemp<desiredTemp){
+  currentTemp++;
+  console.log('The current temperature is now '+ currentTemp + 'F');
+}
+while(currentTemp> desiredTemp){
+  currentTemp--;
+  console.log('The current temperature is now ' + currentTemp + 'F');
+}
+
+// 4. FizzBuzz
+
+for(let i=1; i<=100; i++){
+  if(i%3===0){
+    console.log("Fizz");
+  }
+  if(i%5===0){
+    console.log("Buzz");
+  }
+  if(i%3===0 && i%5===0){
+    console.log("FizzBuzz");
+  } else {
+      console.log(i);
+    }
+}
```

Comment:
double check this. 15 is getting caught in the `fizz` section. 

the main idea here is to put the most specific condition first, as fewer of the numbers will get caught in it. so, lines 49-50 should be first, and the check against 3 and the check against 5 can happen after that